### DISCUSSION TITLE
AWKWARD buying experience

### DISCUSSION THREAD

**Discussion by Anywhere-Music-Player** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#570] [General]
I found this cool app on github, okay. I want to support author and willing to buy a **Pro paid** version.
 - opened site https://apphousekitchen.com/,
 - clicked Get Pro,
 -  Buy NOW!
 - entered e-mail 
 - credit card,
 -  **Purchased**!
 - got e-mail with **transaction receipt**
 
And thats it. Literally, there is no any download link of **dmg** file or any instructions of what to do next, no serial number, nothing.
I just felt like this is some scam or something, I made a purchase and nothing. 
Also, it will be automatically renewed, and I entered credit card info, but I can’t control this, how to remove my credit card info?

Okay, I have a issue, so what I need to do, for sure contact **Support** where I bought it. I buy it.
So I visited https://apphousekitchen.com/support/
```
Premium Support
If you have bought AlDente Pro, you are eligible for our Premium Support. Just check your payment confirmation email which includes our Premium Support email address
```
I opened that e-mail and saw this: 
`If you need help with your AlDente Pro Subscription, please contact us on paddle.net or reply to **this email.**`
First, why now I have to ask support on the **paddle.net** if I bought this on a **apphousekitchen.com**?
Second,  “this email” is just - **help@paddle.com**
I don’t understand this at all, I don’t have any account on that site and I don’t want to use it! 

Overall this is really **AWKWARD.** This should be changed from the scratch, since for now I just want my money back and just use free version!

**Comment by MatthiasKerbl** [comment]
Hi @KAMIKAZEUA,

There was definitely a bug in your buying process. Usually, you get two emails. The first one is the transaction receipt, that's the one you got, and the second one is the one that includes the license code for AlDente Pro, the download link for the AlDente Pro dmg file, and our premium support email address for paying customers.
To solve this, please send me an email to news@apphousekitchen.com with the email address you used to buy our app and I will look into what went wrong and why you did not receive the second email during your payment process, and of course, I will refund your money back!

More answers to your questions:
To cancel your subscription to AlDente Pro (if you bought a subscription) you just have to send us an informal email to cancel it and we will do it immediately. You can find this information on our website in the FAQ section directly under the pricing information.
Why do you see paddle.com everywhere? Paddle is our payment provider. Since it is impossible for us to deal with every payment option and different taxes worldwide, we have to cooperate with another company to offload these tasks. Therefore, if you have any issues regarding the payment, you can write the paddle support.

Sorry for the inconvenience and I hope you understand the situation better now,
Matthias

**Reply by derbalkon** [reply]
Sorry sir, but with all due respect, what he feels is not an isolated case, and I am one of them. From what I experienced, your payment provider service makes people feel that it is **really unreliable** to use.

After I paid, I only received an email saying that my AlDente Pro order was being processed and assuring me that the process would be quick. Then I waited up to 7 hours to receive my license key. In the meantime, I tried to check the progress of my order, and Paddle told me that they could not find my payment information, which made me feel like I was being scammed, and I even came close to calling the bank to intercept this payment. This whole experience is infuriating and makes me feel like I don't want to use it a second time.

So the main reasons I found it uncomfortable were:
- Slow responding
Order processing time and Paddle response time.
- Cumbersome
If you want to unsubscribe, you need to email the customer service and then go through another long wait to get a response.
- Untransparent
There is no clear and visible status check on the subscription and unsubscription process, which makes people feel uncertain.

Please change your order processing process, or at least add a dashboard or something on the website to make the information more transparent.

**Reply by MatthiasKerbl** [reply]
Hi @derbalkon,

Thank you so much for your constructive feedback.
It seems like more people than I thought do not get the seamless purchasing experience that is in place and which I myself always experienced in all my tests. I will try to get these issues fixed.
Furthermore, we will try to implement more transparency in our subscription package.

Cheers,
Matthias

**Comment by Anywhere-Music-Player** [comment]
Yes, I entered some card info, how to remove it? 
I registered 2 licenses on 1 e-mail, how can I check this? 
When purchasing, if I enter wrong email, typo, I’ll not get anything.. 
Support is crazy slow.

**Comment by Anywhere-Music-Player** [comment]
@MatthiasKerbl I just already spending days with that paddle support and it’s again all AWKWARD.
First, their support system is crap, it’s sending me double mails, like when I answer them, it creates new ticket with my response or something, response time is very slow, i’m not getting answers to all my questions, I don’t have any way of controlling my subscriptions, I asked about how to cancel my subscriptions, I have 2 on one e-mail and get just answer- a link to cancel, which just cancel some pro version, but which one? How do I know which? All these questions are endless tickets to them along with their glitchy and slow support system.

I want to use your app, but the way you selling it, the result of final experience given to the customer is really low.
I really dissatisfied as a customer.

**Reply by MatthiasKerbl** [reply]
Hi @KAMIKAZEUA,

I am sorry for your bad support experience. I will contact Paddle to handle situations like that better. Next time, you can write our support and we will try to solve your request as fast as possible.

Thank you that you still want to use our app AlDente!

**Comment by stat11** [comment]
After the purchase, it also showed that the payment was successful, but I did not receive any feedback emails and license codes. The experience was a bit bad. Before purchasing, I highly recommended this app with my friends around me, but now I am ridiculed. Hope to improve the docking of payment providers.

**Comment by Anywhere-Music-Player** [comment]
@MatthiasKerbl recently I found another app I was interested to buy, which uses that paddle site too…
Do you think I brought it? Of course NO. This is a deal breaker now for me and this so pity that more apps trying to use that paddle..
I’ll never ever buy anything on that site.
